man/generate.sh

```diff
@@ -7,6 +7,16 @@ set -eu
 
 mkdir -p ./man/man1
 
+MD2MAN_REPO=github.com/cpuguy83/go-md2man
```

Comment:
And this could be `scripts/docs/generate-man.sh` to be consistent with our other scripts.